setup.py

```diff
@@ -17,26 +13,44 @@ def dependencies(file):
     author_email='manrajsinghgrover@gmail.com',
     url='https://github.com/manrajgrover/halo',
     keywords=[
-        "console",
-        "loading",
-        "indicator",
-        "progress",
-        "cli",
-        "spinner",
-        "spinners",
-        "terminal",
-        "term",
-        "busy",
-        "wait",
-        "idle"
+        'console',
+        'loading',
+        'indicator',
+        'progress',
+        'cli',
+        'spinner',
+        'spinners',
+        'terminal',
+        'term',
+        'busy',
+        'wait',
+        'idle',
+    ],
+    install_requires=[
+        'backports.shutil_get_terminal_size==1.0.0;python_version < "3.3"',
+        'log_symbols==0.0.11',
+        'spinners==0.0.23',
+        'cursor==1.2.0',
+        'termcolor==1.1.0',
+        'colorama==0.3.9',
+        'six==1.11.0',
```

Comment:
Since halo is a library intended to be used by other scripts, it should be liberal in its requirements.
Not pin a version at all, or use '>=' to specify a known minimal version halo needs to function properly.